students/ghassan/lesson05/new_mailroom.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import sys
+
+
+donors = {
+        'Jessie': [100, 300, 500],
+        'James': [343, 22.11],
+        'Ford': [3000, 400]}
+
+
+def main_menu():
+    choice = input(
+        'Pick one:\n1) Send a thank you\n2) Create a Report\n\
+3) Send letters to everyone\n4) Quit\n')
+    return choice
+
+
+def send_thankyou():
+    donor_name = input(
+        'Enter the donor\'s name: (type list if you want to get a list of\
+ donors)\n')
+    if donor_name == 'list':
+        for z in donors.keys():
+            print(z)
+        send_thankyou()
+    try:
+        print('Thank you {} for your generous donation of {}'.format(
+            donor_name, sum(donors[donor_name])))
+    except KeyError:
+        donors[donor_name] = []
+        donation_amount = input('Please enter a donation amount: ')
+        donors[donor_name].append(float(donation_amount))
+        print('Thank you {} for your generous donation of {}'.format(
+            donor_name, sum(donors[donor_name])))
```

Comment:
your mailroom needs to ask for donation amount for existing donors as well.

so if I type in "Ford" it should ask donation amount for Ford and add to DB